Source/Renderer/Context.js

```diff
@@ -673,6 +692,20 @@ define([
             }
         },
 
+        /**
+         * <code>true</code> if the EXT_color_buffer_half_float extension is supported.  This
+         * extension makes the format gl.RGBA16F format color renderable.
+         * @memberof Context.prototype
+         * @type {Boolean}
+         * @see {@link https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_half_float/}
+         * @see {@link https://www.khronos.org/registry/webgl/extensions/EXT_color_buffer_float/}
+         */
+        colorBufferHalfFloat : {
+            get : function() {
+                return (this._webgl2 && this._colorBufferFloat) || (!this._webgl2 && this._colorBufferHalfFloat);
```

Comment:
In WebGL 2, to have half-float color buffer, the color buffer float extension needs to be available?  But in WebGL 1, the color buffer half float needs to be available?